### ISSUE TITLE
industry standard for blocking is to totally block all activity and content

### ISSUE THREAD

**Issue by propertunist** [arckinteractive/block_users#1]
social sites such as G+ and facebook will honor the decision to block a user in all contexts and thus the one who is blocked will not appear in any lists of comments or activity streams once blocked. to facilitate this there is a page provided that contains all the profiles that have been blocked and a means of unblocking them from there if desired. 
it would be great if your plugin here reflected that level of blocking, since at present the activity stream is unaltered when someone is blocked, so if a user wants to stop seeing posts from particular people this plugin cannot presently do that and so for many people it is not very useful.

**Comment by beck24** [comment]
This isn't _really_ our plugin, it was inherited in a project that we took over and the owner wanted to release it.  I don't think we'll be putting any effort into maintaining or updating this.

That said, a good blocking function is needed, but the Elgg core doesn't really have a good global capability for it.  Add in content from additional plugins and specialized data models (relationship vs container) and it gets complicated to the point where non-stock installations would basically require a lot of custom blocking code anyway.

**Comment by propertunist** [comment]
oh ok - so another issue to add to the long list that need to be addressed at a core level. i agree that this type of feature would definitely be best implemented in the core, but since new core features are few and far between (in the previous 3-5 years), i won't hold my breath for it to appear! do you think it is worth creating an issue ticket for it in elgg's github repo?

**Comment by beck24** [comment]
I'd say it's worth getting conversation going.  There is/was some core code in core access queries to start facilitating it but it was never fully enabled.  It could be enabled by plugin and plugins could then handle the UI, I'm just not sure how all-encompassing it would be without really diving into some testing.

**Comment by propertunist** [comment]
ok, i have opened an issue here: https://github.com/Elgg/Elgg/issues/9483
